s2i/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
@jacobsee I think most of the CoP stuff uses the Apache license. We probably want to 1) bump this up to the root of the repo and 2) switch this just to stay consistent.

Reply:
Is it possible in this scenario? This repo includes code originally from vrutkovs/action-s2i, which is MIT licensed...

Reply:
I haven't heard any feedback so I think we'll just move forward with having the Apache license at the root of the repo, but continuing to include the MIT license within this s2i directory unless you have any objections.

Reply:
Nope, that sounds perfect to me!